tasks/coverage/src/driver.rs

```diff
@@ -80,12 +80,12 @@ impl CompilerInterface for Driver {
         ControlFlow::Continue(())
     }
 
-    fn after_semantic(
-        &mut self,
-        program: &mut Program<'_>,
-        ret: &mut SemanticBuilderReturn,
-    ) -> ControlFlow<()> {
+    fn after_semantic(&mut self, ret: &mut SemanticBuilderReturn) -> ControlFlow<()> {
         if self.check_semantic {
+            let AstKind::Program(program) = ret.semantic.nodes().root_node().unwrap().kind() else {
+                // SAFETY: root_node is guaranteed to be Program
+                unsafe { unreachable_unchecked() };
+            };
```

Comment:
`root_node().unwrap()` fails here: https://github.com/oxc-project/oxc/actions/runs/12736474264/job/35496404900?pr=8455#step:7:73

But I am not sure what the correct action should be. Afterall, before my changes program should have existed, so I just assumed it would still exist.

Reply:
Just out of interest, did you figure out what the problem was here?

Reply:
No. Just replaced unwrap with a return statement and all tests passed. Didn't look into the reason why it was needed.

Reply:
I took a look. There is something wrong - that method should never be called if semantic failed, so the unwrap should never fail. But that's a separate issue, and no need to resolve it right now.